### ISSUE TITLE
Priorities

### ISSUE THREAD

**Issue by mj1618** [cadc/glue-stack#13]
Hey Cadell, Matt here.

I like doing a few open source contributions - mind if I take one of your issues? 

Of those 7 issues which ones are the highest priority?

**Comment by mj1618** [comment]
Looks like the two issues in milestone 1.

Have you started on them or not yet?

**Comment by cadc** [comment]
I've started on paging and sorting, but form errors does need doing mainly on the login page for failed authentications and on the signup page for a non unique email.

Otherwise there's also java tests which I'm not that great at. I wanted to do some integration tests from rest endpoint to an in memory database.

Also an admin role would be cool. - someone that can manage users. I was thinking just a column I. The user table only because I have some plans for finer permissions later on.

Search would also be a cool one (not sure if I raised it yet) it would need configuring the querydsl endpoint to handle likes which apparently it can do but I'm not sure how the UI should work in a mobile friendly way.

Living the enthusiasm!
